app/classes/api/model_api.rb

```diff
@@ -149,11 +149,8 @@ def must_have_view_permission!(obj)
     end
 
     def must_have_edit_permission!(obj)
-      if obj.respond_to?(:user_id) &&
-         user.id == obj.user_id
-      elsif obj.respond_to?(:can_edit?) &&
-            obj.can_edit?(user)
-      else
+      unless (obj.respond_to?(:user_id) && user.id == obj.user_id) ||
+             (obj.respond_to?(:can_edit?) && obj.can_edit?(user))
```

Comment:
Fixes Lint/EmptyConditionalBody offense

Reply:
This is probably going to have to be rewritten as guard clauses:

    def must_have_edit_permission!(obj)
      return if obj.respond_to?(:user_id) &&
                user.id == obj.user_id
      return if obj.respond_to?(:can_edit?) &&
                obj.can_edit?(user)
      raise(MustHaveEditPermission.new(obj))
    end

Reply:
No need to re-write, as RuboCop is happy with the long `unless` conditional
(maybe because the re-write is more lines).